src/lib/server/api/discord.ts

```diff
@@ -202,8 +247,26 @@ export async function logResentConfessionViaHttp(
     moderatorId: Snowflake,
     label: string,
     description: string,
+    attachment: Attachment | null,
     botToken = DISCORD_BOT_TOKEN,
 ) {
+    const fields = [
+        {
+            name: 'Authored by',
+            value: `||<@${authorId}>||`,
+            inline: true,
+        },
+        {
+            name: 'Resent by',
+            value: `<@${moderatorId}>`,
+            inline: true,
+        },
+    ];
+
+    if (attachment) {
```

Comment:
Let's be more explicit with the condition that we're checking here.
```suggestion
    if (attachment !== null) {
```